pkgs/development/compilers/rust/binary.nix

```diff
@@ -56,7 +56,7 @@ rec {
     # binaries. The lib.rmeta object inside the ar archive should contain an
     # .rmeta section, but it is removed. Luckily, this doesn't appear to be an
     # issue for Rust builds produced by Nix.
-    dontStrip = stdenv.isDarwin;
+    dontStrip = true;
```

Comment:
Great find. Is there a separate reason to no longer strip on Linux that can be explained in the comment?

Reply:
not sure what exactly happened in this release, I assumed it was something similar to what was said in the comments

Reply:
How big is the size increase?

Reply:
on x86_64-linux, with the patch cherry-picked to master (4eb70719070a576051bad0bcbea9242798cba5e2), checked with `nix path-info -hs`

```
/nix/store/1c5vyn3dvra7jxjq2f9dmcpdcnlb979b-rustc-bootstrap-1.70.0       551.0M
/nix/store/ai9r1rpvqp7scjxq7ima61qim4hv5q9w-rustc-bootstrap-1.70.0       466.2M
```

There is a noticeable increase, but not sure how much it matters since it is already `dontStrip` on darwin, and this is also the boostrap binary

Reply:
If it is only the bootstrap one then it should be alright I guess